en/identity-server/7.0.0/mkdocs.yml

```diff
@@ -791,6 +792,7 @@ nav:
           - SCIM 2.0 Batch operations: apis/scim2/scim2-batch-operations.md
           - SCIM 2.0 Resource types API: apis/scim2/scim2-resource-types.md
           - SCIM 2.0 Service provider configuration API: apis/scim2/scim2-sp-config-rest-api.md
+          - SCIM 2.0 Build user creation payload: apis/scim2/build-scim2-user-creation-payload.md
```

Comment:
IMO, better to move this before           - SCIM 2.0 Patch operations: apis/scim2/scim2-patch-operations.md